### DISCUSSION TITLE
Reflection of a spotlight -- doesn't look right

### DISCUSSION THREAD

**Discussion by buddhasystem** [mitsuba-renderer/mitsuba3#1231] [Q&A] [answered]
Dear All,

A simple scene -- there is a spotlight shining in the direction of the disk, which is tilted by -70 degrees around the vertical axis and is diffuse type of material.
![image](https://github.com/user-attachments/assets/117a0087-ed61-4d7c-9104-adb2f129a1cf)

Since the "spot" emitter produces a cone of light, I wasn't expecting to see a round reflection on the surface of the disk -- which is at an odd angle. What am I getting wrong?

Thank you,
Maxim

**Comment by njroussel** [comment, marked as answer]
Hi @buddhasystem 

Where is the `spot` emitter located? It it co-located with the sensor?

This output seems reasonable to me. If the disk was perpendicular, you'd get a perfect circle (radius determined by the cutoff angle), any rotation will distort that circle. What did you expect?

**Reply by buddhasystem** [reply]
@njroussel 

Thank you! Comments below:

>  Where is the `spot` emitter located? It it co-located with the sensor?

It's at 0.99 in the Z direction, so it's close, and it's looking at 0,0,0.

I think you are right. I'm looking at an ellipse from an angle, and it is projected to me, with my sensor, as a circle. Thank you,
